packages/api/src/command/medical/patient/create-medical-record.ts

```diff
@@ -74,10 +85,40 @@ export async function getMedicalRecordSummary({
   const pdfIsValid = conversionType === "pdf" && pdfCreatedAt;
   const htmlIsValid = conversionType === "html" && htmlCreatedAt;
 
+  let s3BucketName = bucketName;
+  let s3FileKey;
+
+  if (Config.isSandbox()) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    s3BucketName = Config.getSandboxSeedBucketName()!;
+    if (pdfIsValid || htmlIsValid) {
+      const patientName = await getSandboxPatientName(patientId, cxId);
+      s3FileKey = createSandboxMRSummaryFileName(patientName, conversionType);
+      const url = await getSignedURL({ bucketName: s3BucketName, fileName: s3FileKey });
+      return url;
+    }
+  }
+
   if (pdfIsValid || htmlIsValid) {
-    const s3FileKey = createMRSummaryFileName(cxId, patientId, conversionType);
-    const url = await s3Utils.getSignedUrl({ bucketName, fileName: s3FileKey });
+    s3FileKey = createMRSummaryFileName(cxId, patientId, conversionType);
+    const url = await s3Utils.getSignedUrl({ bucketName: s3BucketName, fileName: s3FileKey });
     return url;
   }
   return;
 }
+
+async function getSandboxFileNames(patientId: string, cxId: string): Promise<[string, string]> {
+  const firstName = await getSandboxPatientName(patientId, cxId);
+  const s3HtmlSandboxKey = createSandboxMRSummaryFileName(firstName, "html");
+  const s3PdfSandboxKey = createSandboxMRSummaryFileName(firstName, "pdf");
+  return [s3HtmlSandboxKey, s3PdfSandboxKey];
```

Comment:
Unless using array is better for some specific reason, I suggest returning an object so we avoid inverting the order of elements on the array by mistake.